app/base/forms.py

```diff
@@ -156,3 +155,13 @@ class PropertyForm(FlaskForm):
         "Property location", validators=[DataRequired(), Length(min=5, max=50)]
     )
     submit = SubmitField("Create")
+
+
+class UpdatePropertyForm(CreatePropertyForm):
+    prop_photos = MultipleFileField(
+        "Upload photos of your property",
+        validators=[
+            FileAllowed(["jpeg", "jpg", "png"])
```

Comment:
just curious does it take an absolute file extension like `.jpg` or `jpg` should be enough?

Reply:
No, it supposed to be an extension name like`jpg` without a period